provisioning/JJG-Ansible-Windows/windows.sh

```diff
@@ -17,7 +34,7 @@ YUM=$(which yum 2>/dev/null)
 APT_GET=$(which apt-get 2>/dev/null)
 
 # Make sure Ansible playbook exists.
-if [ ! -f "/vagrant/$ANSIBLE_PLAYBOOK" ]; then
```

Comment:
If you prefer not merging that PR, it's just a matter of changing:

```rb
guest_drupalvm_dir = ENV['DRUPALVM_DIR'] ? "/vagrant/#{ENV['DRUPALVM_DIR']}" : guest_project_dir
```

to

```rb
guest_drupalvm_dir = ENV['DRUPALVM_DIR'] ? ENV['DRUPALVM_DIR'] : ''
```

And then we change it back once Vagrant 1.8.2 comes out.